apps/clinical/src/components/forms/immunization/utils.ts

```
@@ -273,18 +296,33 @@ export function createImmunizationBundleEntries({
+      statusReason: entry.statusReason
+        ? {
+            coding: [
+              {
+                code: entry.statusReason.code,
+                display: entry.statusReason.display,
+              },
+            ],
+          }
         : undefined,
+      route:
+        !isWaiver && entry.route ? { coding: [{ code: entry.route }] } : undefined,
```

ESLint

[error] Replace `·?·{·coding:·[{·code:·entry.route·}]·}` with `⏎··········?·{·coding:·[{·code:·entry.route·}]·}⏎·········`

(prettier/prettier)